app/views/user_journeys/index.erb

```diff
@@ -0,0 +1,19 @@
+<%= content_for(:headerline, 'Moje životné situácie') %>
+<h1 class="govuk-heading-l">Rozpracované životné situácie</h1>
+
+<% if @user_journeys.any? %>
+    <table class="govuk-table">
```

Comment:
Tabulka je taky "programatorsky pohlad", asi by som sa mu snazil vyhnut. Mozno by bolo dobre tam dat normalny zoznam (ten zvykneme balit do two-thirds, aby to nebolo na celu sirku) a ku kazdemu by som dal, ze kedy bola zacata, kolko krokov ma a kolko uz clovek dokoncil. Da sa to vylepsovat do nekonecna, ale tu by som asi nateraz skoncil. 

Ja ked som v uzkych a neviem ako by som to vizualne spravil, tak si pozriem tuto inspiraciu https://tailwindui.com/components#product-application-ui a potom to vyskladam v tom nasom dizajn manuali.

Reply:
@luciajanikova ideme este toto trosku upratat?

Reply:
Jasné, môžeme.